index.js

```diff
@@ -0,0 +1,158 @@
+
+import { NativeModules } from 'react-native';
+import _ from 'lodash';
+
+const { RNShopify } = NativeModules;
+
+const UNKNOWN_ERROR = 'There was an unknown error. Please contact our customer support.';
+
+export default {
+  ...RNShopify,
+  getProducts: (page, collectionId, tags) => {
+    if (collectionId) {
+      return RNShopify.getProductsWithTagsForCollection(page, collectionId, tags);
+    }
+    return tags ? RNShopify.getProductsWithTags(page, tags) : RNShopify.getProductsPage(page);
+  },
+  checkout: (cart) => {
+    return RNShopify.checkout(cart)
+    .catch((error) => {
+      throw new Error(getCheckoutError(error.message, cart));
+    });
+  },
+  setCustomerInformation: (email, customer) => {
+    return RNShopify.setCustomerInformation(email, customer)
+    .catch((error) => {
+      throw new Error(getCheckoutError(error.message));
+    });
+  },
+  completeCheckout: (creditCard) => {
+    return RNShopify.completeCheckout(creditCard)
+    .catch((error) => {
+      throw new Error(getCheckoutError(error.message));
+    });
+  },
+};
+
+/**
+ * Converts checkout errors to a user friendly message. Shopify doesn't have a standard error format.
+ * It replicates the structure that was sent in the request so we need to parse it manually.
+ * This issue was reported here: https://github.com/Shopify/mobile-buy-sdk-ios/issues/22
+ * Here is an example:
+ *
+ * {"errors":{"checkout":{"email":[{"code":"invalid","message":"is invalid","options":{}}],
+ *  "shipping_address":{"country_code":[{"code":"not_supported","message":"is not supported",
+ *  "options":{}}]},"billing_address":{"country_code":[{"code":"not_supported",
+ *  "message":"is not supported","options":{}}]}}}}
+ *
+ * Although Shopify has started to build helper methods for error parsing, there is still work
+ * to be done there. Also, they are different for Android and SDK.
```

Comment:
What does it mean `for android and sdk`? Maybe you wanted to say `for android and ios`?